website/redirects.js

```diff
@@ -130,13 +130,7 @@ module.exports = [
       "/terraform/cloud-docs/integrations/service-now/service-catalog-terraform/example-customizations",
     permanent: true,
   },
-  {
-    source: "/terraform/cloud-docs/integrations/service-now/service-now-v1",
```

Comment:
We are confident that no one is using v1 anymore. It's been several years since it was removed from the ServiceNow Store. We should be good just deleting v1 completely without redirects.